tasks/README.md

```diff
@@ -0,0 +1,66 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<end date>",
+  "startedOn":"<start date>",
+  "status": "Active",
+  "ownerId":"<app owner user id>",
+  "percentCompleted":10%,
+  "dependsOn": [
+    <tasksList>
+  ],
+  "participants": [list of users participating in this tasks. For 'group' type only],
+  "completionAward": { gold: 3, bronze: 300 },
+  "lossRate": { gold: 1 }  // Loss per day of overshoot after deadline
+}
+```
+
+## **Requests**
+
+|               Route                |    Description    |
+| :--------------------------------: | :---------------: |
+|      [GET /tasks](#get-tasks)      | Returns all tasks |
+|     [POST /tasks](#post-tasks)     | Creates new task  |
+| [PATCH /tasks/:id](#patch-tasksid) |   Updates tasks   |
+
+## **GET /tasks**
+
+> returns all the tasks
+
+## Response
+
+```
+[
+  {<task_object>},
+  {<task_object>}
+]
+
+```
+
+## **POST /tasks**
+
+## Body
+
+```
+{<task_object>}
+```
+
+## Response
+
+Return the same response from [GET /tasks](#get-tasks)
+
+## **PATCH /tasks/:id**
+
+## Body
+
+```
+{<task_object>}
+```
```

Comment:
Nice work! @sumitd94  🎉 

I would like to suggest something here. For this `PATCH /tasks/:id` we can mention the below given thing in contract of the route.
`Params
Required: id=[string]` 

You can refer this : https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/users. This way we can have similarity in every file. 😄 

Thank you!!

Reply:
Sure Thanks, Will update it .